src/ex7/client/src/App.js

```diff
@@ -27,31 +30,74 @@ function App() {
     
   }, []);
 
+  const addItemToTodoList = async (todoInput) => {
+    let todoListAfterAdd=await itemClient.addTodo(todoInput);
+    setTodoList(todoListAfterAdd);
```

Comment:
you shouldn't return all todolist from the server everytime, and the action should add the only new item